object.go

```diff
@@ -0,0 +1,1131 @@
+	ExternalUrl  string `graphql:"externalUrl" json:"externalUrl"`   // The external URL of the infrastructure resource in its provider.
+	ProviderName string `graphql:"providerName" json:"providerName"` // The name of the provider (e.g. AWS, GCP, Azure).
+}
+
+// InfrastructureResourceSchema represents a schema for Infrastructure Resources.
+type InfrastructureResourceSchema struct {
+	Schema JSON   `graphql:"schema" json:"schema" scalar:"true"` // The schema definition.
+	Type   string `graphql:"type" json:"type"`                   // The schema type.
+}
+
+// IssueTrackingIntegration represents .
+type IssueTrackingIntegration struct {
+	CreatedAt   iso8601.Time `graphql:"createdAt" json:"createdAt"`     // The time this integration was created.
+	Id          ID           `graphql:"id" json:"id"`                   // The unique identifier for the integration.
+	InstalledAt iso8601.Time `graphql:"installedAt" json:"installedAt"` // The time that this integration was successfully installed, if null, this indicates the integration was not completed installed.
+	Name        string       `graphql:"name" json:"name"`               // The name of the integration.
+	Type        string       `graphql:"type" json:"type"`               // The type of the integration.
+}
+
+// JenkinsIntegration represents .
+type JenkinsIntegration struct {
+	CreatedAt   iso8601.Time `graphql:"createdAt" json:"createdAt"`     // The time this integration was created.
+	Id          ID           `graphql:"id" json:"id"`                   // The unique identifier for the integration.
+	InstalledAt iso8601.Time `graphql:"installedAt" json:"installedAt"` // The time that this integration was successfully installed, if null, this indicates the integration was not completed installed.
+	Name        string       `graphql:"name" json:"name"`               // The name of the integration.
+	Type        string       `graphql:"type" json:"type"`               // The type of the integration.
+}
+
+// KubernetesIntegration represents .
+type KubernetesIntegration struct {
+	CreatedAt   iso8601.Time `graphql:"createdAt" json:"createdAt"`     // The time this integration was created.
+	Id          ID           `graphql:"id" json:"id"`                   // The unique identifier for the integration.
+	InstalledAt iso8601.Time `graphql:"installedAt" json:"installedAt"` // The time that this integration was successfully installed, if null, this indicates the integration was not completed installed.
+	Name        string       `graphql:"name" json:"name"`               // The name of the integration.
+	Type        string       `graphql:"type" json:"type"`               // The type of the integration.
+}
+
+// Language represents a language that can be assigned to a repository.
+type Language struct {
+	Name  string  `graphql:"name" json:"name"`   // The name of the language.
+	Usage float32 `graphql:"usage" json:"usage"` // The percentage of the code written in that language.
```

Comment:
do we know if Ruby is float64 or float32?  Honestly if it works float64 is better in golang because the standard math lib works with float64 and you have to bring in a specialized math32 external library to support math operations on float32 OR you have to convert / cast type all the time.